wellcomecollection/wellcomecollection.org
# Remove unnecessary redirects

## What does this change?

Related to #11300 & [#11280](https://github.com/wellcomecollection/wellcomecollection.org/issues/11280)

The first commit was just an alphabetical reorder for readability so I'd suggest reviewing this commit by commit for clarity. I aimed to have clear commit messages for all.

I've modified a lot of UIDs in Prismic to then be able to delete a lot of redirects (matched their vanity URL in Prismic).

I've then moved them all to target UID based URLs.

This removes about half the redirections 🎉 

## How to test

You can go confirm my theories about each of them, but we can't actually test this until `Terraform apply` gets run, which I'll do once this is in staging. We can then test them all on there and when happy, apply the changes to production as well.

## How can we measure success?

A clearer, more useful list of redirections, made minimal and readable!

## Have we considered potential risks?

As we can test on staging before, I think this is low risk.
